rust/datafusion/src/execution/context.rs

```diff
@@ -210,6 +216,103 @@ impl ExecutionContext {
         Ok(plan)
     }
 
+    /// Create a physical plan from a logical plan
+    pub fn create_physical_plan(
+        &mut self,
+        logical_plan: &Arc<LogicalPlan>,
+    ) -> Result<Arc<dyn ExecutionPlan>> {
+        match logical_plan.as_ref() {
+            LogicalPlan::TableScan {
+                table_name,
+                projection,
+                ..
+            } => match (*self.datasources).borrow().get(table_name) {
+                Some(provider) => {
+                    let partitions = provider.scan(projection, 16 * 1024)?;
```

Comment:
What's the rationale for hard coding `16 * 1024`?

Reply:
I've made this a parameter now.